Run the tests on Android Chrome

mithro opened the pull request.

mithro committed:
Getting android working.

alancutter commented:
Looks like we might need to drop our accuracy to two decimal places...

dstoc commented on run-tests-android.sh:
```diff
@@ -0,0 +1,72 @@
+  if kill -0 $EMULATOR_PID; then
+    break
+  else
+    echo "Emulator has crashed, try to start it again..."
```
try -> trying?

dstoc commented on test/test-runner.html:
```diff
@@ -171,7 +171,11 @@ <h1 id=status>Pending</h1>
  * @return {number} Time as this very moment.
  */
 function now() {
-  return performance.now();
```
this seems unrelated to running the tests on android

dstoc commented on .travis-setup.sh:
```diff
@@ -7,7 +7,7 @@ ls -l /dev/shm
 ls -l /dev/shm
 
-sudo apt-get update --fix-missing
+sudo apt-get update --fix-missing > /dev/null 2>&1
```
-qq ?

mithro committed:
Merge remote-tracking branch 'up/master' into android

Conflicts:
	.travis-setup.sh

mithro committed:
Updating Android Developer toolkit and Chrome Driver.

mithro committed:
Adding intent to call on package.

mithro committed:
Bail if non-emulator Android devices are detected.

mithro committed:
Better detection of emulator crashing.

mithro committed:
Split the test run into two parts so it completes.

mithro committed:
Create cache buster at test-runner level and pass it down.

This means that 100 unique copies of web-animation.js (and similar files) are
not loaded when running inside the test-runner (a big performance problem for
Android).

mithro committed:
Temp.
